gear/gear/session.py

```diff
@@ -21,7 +19,8 @@ def setup_aiohttp_session(app):
                 secure=True,
                 httponly=True,
                 samesite='Lax',
-                domain=os.environ['HAIL_DOMAIN'],
+                domain=deploy_config._domain,
+                path=deploy_config._base_path or '/',
```

Comment:
Setting the base path for the cookie should allow us to access multiple dev namespaces without the cookies interfering with each other. Were we just having to log in again when accessing someone else's dev namespace?